db/migrate/20220503132944_change_schedule_milestones_index.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ChangeScheduleMilestonesIndex < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    remove_index :schedule_milestones, column: %i[schedule_id milestone_id declaration_type], name: :schedules_milestones_schedule_milestone_declaration_type
+    remove_index :schedule_milestones, column: %i[milestone_id schedule_id declaration_type], name: :milestones_schedules_schedule_milestone_declaration_type
+
+    add_index :schedule_milestones, %i[schedule_id milestone_id], algorithm: :concurrently
+
+    add_index :schedule_milestones, %i[milestone_id schedule_id], algorithm: :concurrently
+  end
+end
```

Comment:
I take it this does the up/down without issue

Reply:
tested it and yes it seems fine

Reply:
👍